amundsen_application/static/js/components/Alert/index.tsx

```diff
@@ -2,16 +2,23 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import * as React from 'react';
+import SanitizedHTML from 'react-sanitized-html';
 
 import { IconSizes } from 'interfaces';
-import { AlertIcon } from '../SVGIcons';
+import { NoticeSeverity } from 'config/config-types';
+import { AlertIcon, InformationIcon } from '../SVGIcons';
 
 import './styles.scss';
 
-const STROKE_COLOR = '#b8072c'; // $red70
+const SEVERITY_TO_COLOR_MAP = {
+  [NoticeSeverity.INFO]: '#3a97d3', // cyan50
```

Comment:
We don't have these colors on the _colors css file as variables?

Reply:
Yeah, that's the 'cyan50', but we cannot bring the scss variables into the JS files :(